test.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
```

Comment:
These aren't "unit tests".  Unit tests are intended to be able to be run very quickly, and each test only a single piece of your application (say, a single class or a single method).  They are intended to test a bunch of different cases that would be difficult for you to manually test.